.gitignore

```diff
@@ -1,5 +1,4 @@
 *temp*
-assets/
```

Comment:
No problem at all. As usual, your contributions are awesome and appreciated.
On May 11, 2016 6:52 PM, "Kale Blankenship" <notifications@github.com>
wrote:

> In .gitignore
> <https://github.com/Mierdin/todd/pull/22#discussion_r62955212>:
>
> > @@ -1,5 +1,4 @@
> >  *temp*
> > -assets/
>
> Sure thing, not sure I'll get a chance to updating the PR tonight, but
> I'll get to it soon.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/Mierdin/todd/pull/22/files/d451b31ecbb8154646180e52c0d3bfb9903df362#r62955212>
>